libcalico-go/lib/namespace/resource.go

```diff
@@ -25,6 +25,7 @@ const (
 	// to avoid an import loop.
 	KindKubernetesNetworkPolicy = "KubernetesNetworkPolicy"
 	KindKubernetesEndpointSlice = "KubernetesEndpointSlice"
+	KindK8sService              = "K8sService"
```

Comment:
`KindKubernetesService` and `KubernetesService` to be consistent with other values here, please :grin: